extensions/view/examples/example-landsat8.json

```diff
@@ -0,0 +1,86 @@
+{
+    "stac_version": "0.9.0-rc1",
+    "stac_extensions": [
+      "sat",
+      "view"
+    ],
+    "id": "LC08_L1TP_107018_20181001",
+    "collection": "landsat-8-l1",
+    "type": "Feature",
+    "bbox": [
+        148.13933,
+        59.51584,
+        152.52758,
+        60.63437
+    ],
+    "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+            [
+                [
+                    152.52758,
+                    60.63437
+                ],
+                [
+                    149.1755,
+                    61.19016
+                ],
+                [
+                    148.13933,
+                    59.51584
+                ],
+                [
+                    151.33786,
+                    58.97792
+                ],
+                [
+                    152.52758,
+                    60.63437
+                ]
+            ]
+        ]
+    },
+    "properties": {
+        "datetime": "2018-10-01T01:08:32.033Z",
+        "view:sun_azimuth": 168.8989761,
```

Comment:
There are two of the five fields missing in the example. Would be great if the example shows all potential fields.

Reply:
on it.

Reply:
https://github.com/radiantearth/stac-spec/pull/734/commits/e9ef31cac4817e3b622044fbf16233453c4909ab

Reply:
Great, and nicely ordered :+1